### ISSUE TITLE
Fixed point photography -- app labelling of groups

### ISSUE THREAD

**Issue by sacrevert** [NERC-CEH/dunes-app#55]
The app provides "Transect" as an option under the Fixed point photography Location screen, however, according to how these groups are set-up on the website, this should be labelled "Group" rather than "Transect"
@kazlauskis @LaurenceJonesES 

Note that this deals with the first bullet in the Word doc under #54

**Comment by johnvanbreda** [comment]
@DavidRoy app issue?

**Comment by DavidRoy** [comment]
@johnvanbreda you are right I think but maybe implications for the API reports used by the app?

**Comment by johnvanbreda** [comment]
I'm not sure I follow why the API reports are implicated but @kazlauskis can let me know if that is the case.

**Comment by kazlauskis** [comment]
Yeah, the terminology isn't all the way consistent between the website, manual and the app. In the manual, we don't use the word 'groups' either but mention that the points can be along a transect. If this is about changing the 'Transect' to 'Group' in this page then that should be easy though it would require some re-translation to Welsh. 

EDIT: for the reports that the app use - this shouldn't require John changing the report as long as the data exported uses the same naming.

<img width="395" alt="Screenshot 2020-11-16 at 16 25 16" src="https://user-images.githubusercontent.com/7383162/99279670-67310200-2828-11eb-9dcf-633a3adfb286.png">

**Comment by DavidRoy** [comment]
@LaurenceJonesES to confirm

**Comment by LaurenceJonesES** [comment]
I think we need a consistent &/or flexible approach that works across all activities, and is consistent across 'set up' on the website, use in the app, and naming conventions for downloaded data.
-Simplest would be to call this 'Transect/Group' for all activities. If its not too much work, then it could be called 'Transect' for the two transect activities: Profile and Zonation, and 'Group' for all others.
-Note that group may not always be defined, so in the app (& possibly the download), it might need a default which appears such as [not assigned].  This could perhaps be applied as the default during set up, which carries through to app and downloads unless a Transect/Group name is added.
